src/WebJobs.Script.WebHost/Management/TriggersPayload.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public sealed class TriggersPayload : TriggersOperationResult
```

Comment:
Name this class GetTriggersResult - I thought we had already agreed on that based on your other comments? It's not just a payload, it's also a result

Reply:
But then Shyju asked to remove "Get" from the name. How about TriggersResult? or I'm fine with GetTriggersResult as well.

Reply:
Sure, TriggersResult is fine :)

Reply:
Changed to TriggersResult